engine/src/test/java/com/redhat/cloud/notifications/processors/email/EmailTest.java

```diff
@@ -54,23 +55,22 @@
 @QuarkusTest
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @QuarkusTestResource(TestLifecycleManager.class)
-public class EmailTest {
+public class EmailTest extends DbIsolatedTest {
```

Comment:
how do we deal with those tests in engine?

Reply:
`DbIsolatedTest` is gone in `engine`. You can simply remove it from the test.

Reply:
done